test/data/test_feature_store.py

```diff
@@ -51,6 +50,17 @@ def _remove_tensor(self, attr: TensorAttr) -> bool:
         del self.store[MyFeatureStore.key(attr)]
         return True
 
+    def _get_tensor_size(self, attr: TensorAttr):
+        attr.index = None
```

Comment:
Why do we need to set it to `None`. If `index` is set, I would assume that we would return the `size` of the filtered data.